e2e/test/helpers/TestDeviceCallbackHandler.cs

```diff
@@ -95,22 +100,23 @@ internal async Task WaitForMethodCallbackAsync(CancellationToken ct)
             _methodExceptionDispatch?.Throw();
         }
 
-        internal async Task SetTwinPropertyUpdateCallbackHandlerAsync(string expectedPropName)
+        internal async Task SetTwinPropertyUpdateCallbackHandlerAndProcessAsync<T>(string expectedPropName, T expectedPropValue)
```

Comment:
I haven't made the similar change to this yet, but it'll follow in a subsequent PR.